ui/src/app/workflows/components/workflow-details/workflow-details.tsx

```diff
@@ -263,6 +270,7 @@ export const WorkflowDetails = ({history, location, match}: RouteComponentProps<
                     {parsedSidePanel.type === 'logs' && (
                         <WorkflowLogsViewer workflow={workflow} nodeId={parsedSidePanel.nodeId} container={parsedSidePanel.container} archived={false} />
                     )}
+                    {parsedSidePanel.type === 'events' && <EventsPanel namespace={namespace} kind='Pod' name={parsedSidePanel.nodeId} />}
```

Comment:
Node events now appear is a much more spacious side-panel:

![image](https://user-images.githubusercontent.com/1142830/105527641-d26bda00-5c98-11eb-9d6f-5b7e49d670d7.png)